fortanix-vme/aws-nitro-enclaves/eif-tools/Cargo.toml

```diff
@@ -22,7 +22,7 @@ env_logger = "0.9"
 log = "0.4"
 nitro-cli = { git = "https://github.com/fortanix/aws-nitro-enclaves-cli.git", branch = "main" }
 once_cell = "1.9.0"
-serde = { version = "1.0", features = ["derive"] }
+serde = { version = "1.0.214", features = ["derive"] }
```

Comment:
Is this necessary? Is there any dependency needs this?
Otherwise, you could just update the lock file and leave all these changes to `serde` in Cargo.toml untouched.